schema@v1.3.0/dataset.json

```diff
@@ -0,0 +1,144 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.data.amsterdam.nl/dataset@v1.3.0",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "./schema@v1.3.0#/definitions/basicProperties"
+    }
+  ],
+  "required": [
```

Comment:
"oneOf": [
        {
            "required": [
    "status",
    "creator",
    "authorizationGrantor",
    "owner",
    "publisher",
    "auth",
   "objectstore"
            ]
        },
        {
            "required": [
    "status",
    "creator",
    "authorizationGrantor",
    "owner",
    "publisher",
    "auth",
   "tables"
            ]
        },

Reply:
Je zou dit nog kunnen doen. Dan moet een dataset minimaal een objectstore of een tabel hebben